### PR TITLE
useAtomCallback tests

### PR DESCRIPTION
**Pull request by Aslemammad**
:tada: I made the tests, Any suggestions, ....

### CODE REVIEW

File: tests/utils/useAtomCallback.test.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      useCallback((get) => {
+        const currentCount = get(countAtom)
+
+        setSecondCount(currentCount)
+        return currentCount
+      }, [])
+    )
+    useEffect(() => {
+      const read = async () => await readCount()
+      read()
```
Comment (dai-shi) [inline tests/utils/useAtomCallback.test.tsx:22]:
```suggestion
      readCount()
```

We could simplify this?

File: tests/utils/useAtomCallback.test.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
+import React, { useCallback, useEffect, useState } from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import { Provider, useAtom, atom } from '../../src/index'
+import { useAtomCallback } from '../../src/utils'
+
+it('useAtomCallback with get', async () => {
+  const countAtom = atom(0)
+
+  const Parent: React.FC = () => {
+    const [count, setCount] = useAtom(countAtom)
```
Comment (dai-shi) [inline tests/utils/useAtomCallback.test.tsx:10]: It would be better, we we didn't have this here, because that should be the purpose of useAtomCallback.

**Review by dai-shi — COMMENTED** [review]
Added some comments on `with get` test.

File: tests/utils/useAtomCallback.test.tsx (hunk 2 above)
Comment (Aslemammad) [reply tests/utils/useAtomCallback.test.tsx:10]: Sorry, which line you mean?

Comment (dai-shi) [reply tests/utils/useAtomCallback.test.tsx:10]: L10 `useAtom(countAtom)` wouldn't be in `Parent`.

**Comment by Aslemammad** [comment]
@dai-shi How about now.

**Comment by dai-shi** [comment]
LGTM
Would you like to try using waitFor instead of multiple findByText? cf. https://github.com/pmndrs/jotai/issues/88#issuecomment-725329174

**Comment by Aslemammad** [comment]
@dai-shi I uses them now, What's your idea now?

**Review by dai-shi — APPROVED** [review]

**Comment by dai-shi** [comment]
Looks good. Merging this.

**Comment by Aslemammad** [comment]
Thanks dai-shi.

**Pull request merged**